core/test/surplus-test.ts

```diff
@@ -42,8 +42,10 @@ describe('Surplus Auction', () => {
     it('fetches active auctions', async () => {
         const auctions = await fetchActiveSurplusAuctions('custom');
         expect(auctions.length).to.equal(5);
-        expect(auctions[0].id).to.equal(2328);
-        expect(auctions[0].state).to.equal('have-bids');
+        const auction = auctions[0] as SurplusAuctionActive;
+        expect(auction.id).to.equal(2328);
+        expect(auction.state).to.equal('have-bids');
+        expect(auction.auctionStartDate.toISOString()).to.equal('2022-01-26T01:31:58.000Z');
```

Comment:
Do you want to at least compare that end date is later than the start date? (cause this number doesn't make any meaning to the reader)